src/tools/roc_recv/main.cpp

```diff
@@ -17,13 +18,16 @@
 #include "roc_pipeline/parse_port.h"
 #include "roc_pipeline/receiver.h"
 #include "roc_sndio/backend_dispatcher.h"
+#include "roc_sndio/driver_info.h"
 #include "roc_sndio/pump.h"
 
 #include "roc_recv/cmdline.h"
 
 using namespace roc;
 
 int main(int argc, char** argv) {
+    core::HeapAllocator allocator;
```

Comment:
My comment "move it to the top" was a bit misleading, sorry. It should be before `if (args.list_drivers_given) {` but after `crash_handler` and after `core::Logger::instance().set_level`. These two things should be before all other stuff.